app/src/main/java/org/mozilla/focus/contextmenu/ContextMenuCandidates.kt

```diff
@@ -22,8 +22,9 @@ object ContextMenuCandidates {
         appLinksUseCases: AppLinksUseCases,
         snackBarParentView: View,
         snackbarDelegate: ContextMenuCandidate.SnackbarDelegate = DefaultSnackbarDelegate()
-    ): List<ContextMenuCandidate> =
-        if (FocusNimbus.features.tabs.value().isMultiTab) {
+    ): List<ContextMenuCandidate> {
+        val multiTabExperiment = FocusNimbus.features.tabs.value()
```

Comment:
Nit: the `value()` is a config, not an experiment. Suggest renaming to `tabConfig` or `multiTabConfg`.